app/Http/Controllers/Auth/RegisterController.php

```diff
@@ -56,26 +65,34 @@ protected function validator(array $data)
         return Validator::make($data, [
             'name' => 'required|max:255',
             'email' => 'required|email|max:255|unique:users',
+            'airline' => 'required',
+            'home_airport' => 'required',
             'password' => 'required|min:5|confirmed',
         ]);
     }
 
-    /**
-     * Create a new user instance after a valid registration.
-     *
-     * @param  array  $data
-     * @return User
-     */
-    protected function create(array $data)
+    public function register()
     {
-        $user = User::create([
-            'name' => $data['name'],
-            'email' => $data['email'],
-            'password' => bcrypt($data['password']),
+        // Validate
+        $this->validate(request(), [
+            'name' => 'required',
+            'email' => 'required|email',
+            'airline' => 'required',
+            'home_airport' => 'required',
+            'password' => 'required|confirmed'
         ]);
 
-        $role = Role::where('name', 'admin')->first();
+        # TODO: I'm just feeling we need to do something with Ranking? I forgot.
```

Comment:
Let's move this into the `Services/PilotService.php` class. After creating the user, you can call `calculatePilotRank($user)` in the same class